packages/web/src/scss/tools/_form-fields.scss

```diff
@@ -32,6 +32,7 @@
 
     display: block;
     margin-top: form-fields-theme.$message-margin-top;
+    list-style: none;
```

Comment:
How about an unwanted bottom margin of `<ul>`? Isn't there one that needs to be reset?

Reply:
Looks OK. The bottom margin is only on `ul:not(:last-child)`.